MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -183,10 +184,12 @@ - (void)start:(NSString *)appSecret withServices:(NSArray<Class> *)services {
     if (configured) {
 
       NSArray *sortedServices = [self sortServices:services];
+      NSMutableArray<NSString*>* servicesNames = [NSMutableArray arrayWithCapacity:sortedServices.count];
 
       for (Class service in sortedServices) {
-        [self startService:service];
+        [servicesNames addObject:[self startService:service].storageKey];
```

Comment:
@jaelim-ms we already have a valid field for this. Why not use `[service logTag];`?